testnet-contracts/truffle.js

```diff
@@ -1,3 +1,7 @@
+require("dotenv").config();
+
+var HDWalletProvider = require("@truffle/hdwallet-provider");
+
 module.exports = {
   networks: {
     development: {
```

Comment:
should be develop, not development in order to work with new version of truffle

Reply:
https://github.com/swishlabsco/peggy/pull/2/commits/df7eceafe4285734bfd160fc9f1bdfb52ae973fd